PhysicsTools/Heppy/python/physicsutils/JetReCalibrator.py

```diff
@@ -137,8 +137,8 @@ def getCorrection(self,jet,rho,delta=0,corrector=None,uncertainty="Total", recal
         JetUncertainty.setJetEta(jet.eta())
         JetUncertainty.setJetPt(corr * jet.pt() * jet.rawFactor())
 
-        try:
-            jet.jetEnergyCorrUncertainty = self.JetUncertainty.getUncertainty(True)
```

Comment:
right that was my quick hack this morning to get thing running, should not be needed with jpata fixes